readme.md

```diff
@@ -161,6 +161,17 @@ antibody bundle mafredri/zsh-async
 antibody bundle sindresorhus/pure
 ```
 
+## Shell Ports
```

Comment:
`Shell` here is superfluous (and could be understood as pure is not for the shell), pure is a shell theme and thus these are just *ports*.